### ISSUE TITLE
SER# requested for smartshunt on serial interface

### ISSUE THREAD

**Issue by niecore** [birdie1/victron#4]
Hello, 

I got following crash:

```
(venv) pi@rpi-jinx:~/victron $ python victron.py -d 0
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 1266, in run
    self.function(*self.args, **self.kwargs)
  File "/home/pi/victron/victron.py", line 25, in victron_thread
    v = Victron(config, vdevice_config, output, args, thread_count, thread_q)
  File "/home/pi/victron/lib/victron.py", line 40, in __init__
    pid, ser, fw = self.victron_type.get_device_info()
  File "/home/pi/victron/lib/victron_serial/victron_serial.py", line 26, in get_device_info
    return self.victron_device.get_device_info()
  File "/home/pi/victron/lib/victron_serial/victron_smartshunt.py", line 61, in get_device_info
    ser = self.MAP['SER#'][4](data['SER#'], self.MAP['SER#'])
KeyError: 'SER#
```

```
devices:
    - name: Shunt1
      type: smartshunt
      protocol: serial
      port: /dev/ttyUSB0
```
It seems like the smartshunt is not supporting this field on the serial interface.

**Comment by fti7** [comment]
I have the same issue (Latest Smartshunt Firmware via Serial)

**Comment by birdie1** [comment]
Fixed with commit 06d75538ff4b6b3ede639f96f4c0634f5d16fe6b
